docs/guides/cluster.mdx

```diff
@@ -22,20 +22,29 @@ This guide shows how to deploy a distributed Restate cluster consisting of 3 nod
 To deploy a 3 node distributed Restate cluster, copy the `docker-compose.yml` and run `docker compose up`.
 
 ```yaml
+# Common environment variables for all restate nodes
 x-environment: &default-env
+  # To customize logging, check https://docs.restate.dev/operate/monitoring/logging
+  RESTATE_LOG_FILTER: "restate=info"
```

Comment:
I think I preferred the flow of the original property order here :-) Cluster level -> node level defaults. They also roughly follow the section ordering in the server config schema making it easier to look up for reference.

Reply:
My rationale here was more "highest likelihood to be changed"